docs/avdanos/contributing/development/code_guidelines/rust_guidelines.mdx

```diff
@@ -88,6 +88,22 @@ let var: f64 = 7.42;
 let var = 7.42
 ```
 
+#### When printing a simple variable value don't do the traditional way. Why? Because it's less write time
```

Comment:
~~Because it's less write time~~

It is concise, organised, and right to the point

Reply:
Changing it